luatest/process.lua

```diff
@@ -17,9 +18,15 @@ local function to_const_char(input)
     return ffi.cast('char *const*', result)
 end
 
--- Starts process and returns immediately, not waiting until process is finished.
+--- Starts process and returns immediately, not waiting until process is finished.
+-- @param path Executable path.
+-- @param[opt] args
+-- @param[opt] env
+-- @param[opt] options
+-- @param[opt] options.chdir Directory to chdir into before starting process.
+-- @param[opt] options.ignore_gc Don't install handler which kills GC'ed processes.
 function Process:start(path, args, env, options)
-    checks('table', 'string', '?table', '?table', {chdir = '?string'})
+    checks('table', 'string', '?table', '?table', {chdir = '?string', ignore_gc = '?boolean'})
```

Comment:
How can it be used?

Reply:
`{ignore_gc = true}` ?
This option:
- provides control over very specific feature,
- documents code, and make it easier to understand what's going on in  the code.